nixos/tests/sudo.nix

```diff
@@ -42,38 +46,70 @@ in
 
             # CONFIGURATION FOR TEST CASES
             { users = [ "test1" ]; groups = [ "foobar" ]; commands = [ "ALL" ]; }
-            { groups = [ "barfoo" 1337 ]; commands = [ { command = "ALL"; options = [ "NOPASSWD" "NOSETENV" ]; } ]; }
-            { users = [ "test5" ]; commands = [ { command = "ALL"; options = [ "NOPASSWD" "SETENV" ]; } ]; runAs = "test1:barfoo"; }
+            { groups = [ "barfoo" 1337 ]; commands = [ { command = "ALL"; options = [ "NOPASSWD" ]; } ]; }
+            { users = [ "test5" ]; commands = [ { command = "ALL"; options = [ "NOPASSWD" ]; } ]; runAs = "test1:barfoo"; }
           ];
         };
       };
 
     nodes.strict = { ... }: {
+      environment.systemPackages = [ pkgs.faketty ];
```

Comment:
-1 from me on using faketty in nixos sudo tests. sudo has a bunch of complex logic regarding PTYs and I'm not confident this will be properly exercised anymore with faketty.

IMO if sudo-rs requires it, that's a bug in sudo-rs.